scripts/tests/test_generate_decoding_fuzz_corpus.py

```
@@ -53,6 +61,50 @@ def test_generated_phase29_contract_is_bound_to_phase30_boundaries(self) -> None
+            with self.assertRaises(ValueError):
+                GEN.load_phase113_fixture(root)
+
+    def test_load_phase113_fixture_invalid_json_raises(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp:
+            root = pathlib.Path(tmp)
+            raw = b"{not-json"
+            self.write_phase113_source(root, raw)
+            original = GEN.PHASE113_SOURCE_SHA256
+            GEN.PHASE113_SOURCE_SHA256 = hashlib.sha256(raw).hexdigest()
+            try:
+                with self.assertRaises(ValueError):
```

Ruff (0.15.10)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)